pkg/sloop/queries/respayloadquery.go

```diff
@@ -105,7 +102,7 @@ func GetResPayload(params url.Values, t typed.Tables, startTime time.Time, endTi
 	return bytes, nil
 }
 
-func getSeekKey(keyComparator *typed.WatchTableKey, startTime time.Time) *typed.WatchTableKey {
+func GetSeekKey(keyComparator *typed.WatchTableKey, startTime time.Time) *typed.WatchTableKey {
```

Comment:
this change to global is for the other change right?

Reply:
yes moved it from this change. Thanks for pointing it out.